### PR TITLE
North West 4 - Morteza Khojasteh - JavaScript - week 1(exercises)

### PR DESCRIPTION
**Pull request by Morteza-Khojasteh**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: exercises/F-strings-methods/exercise2.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
-const name = " Daniel  ";
+const names = "Morteza   ";
+const nameLength = (names.trim()).length;
```
Comment (lannem) [inline exercises/F-strings-methods/exercise2.js:2]: nice combining of these 2 methods in 1 line 🙂

File: exercises/J-functions/exercise.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 function halve(number) {
   // complete the function here
+  const halveNumber = number / 2;
+  return halveNumber;
```
Comment (lannem) [inline exercises/J-functions/exercise.js:4]: If you wanted to reduce the number of lines you could return the expression without staring it in a variable: `return number/2`

**Review by lannem — COMMENTED** [review]
great work Morteza well done :)

File: exercises/F-strings-methods/exercise2.js (hunk 1 above)
Comment (Morteza-Khojasteh) [reply exercises/F-strings-methods/exercise2.js:2]: Thanks

File: exercises/J-functions/exercise.js (hunk 2 above)
Comment (Morteza-Khojasteh) [reply exercises/J-functions/exercise.js:4]: Yeah, I just tried to write code clear

**Pull request closed without merging**